addons/http_routing/models/ir_http.py

```diff
@@ -334,7 +334,7 @@ def _get_translation_frontend_modules_name(cls):
 
     @classmethod
     def _get_frontend_langs(cls):
-        return [code for code, _ in request.env['res.lang'].get_installed()]
+        return request.env['res.lang'].get_frontend()
```

Comment:
Shouldn't we cast this to a `list` ?

Reply:
keeping it `{code: LangData}` is easier to use
1. to check if `'fr_FR' in _get_frontend_langs()`
2. and get `_get_frontend_langs['fr_FR'].url_code`